installers/unix.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+zip_url=https://github.com/tomasklaen/uosc/releases/latest/download/uosc.zip
+conf_url=https://github.com/tomasklaen/uosc/releases/latest/download/uosc.conf
+zip_file=/tmp/uosc.zip
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+cleanup() {
+	if [ -f "$zip_file" ]; then
+		echo "Deleting: $zip_file"
+		rm -f $zip_file
+	fi
+}
+
+abort() {
+	cleanup
+	printf "%s\n" "$@" >&2
+	exit 1
+}
+
+# Check OS
+OS="$(uname)"
+if [ "${OS}" == "Linux" ]; then
+	data_dir="${XDG_CONFIG_HOME:-~/.config}/mpv"
```

Comment:
```suggestion
	data_dir="${XDG_CONFIG_HOME:-$HOME/.config}/mpv"
```
`~` doesn't get automatically expanded